e2e/cypress/integration/messaging/direct_messages_spec.js

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @messaging
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+import users from '../../fixtures/users.json';
+
+const userActive = users['user-1'];
+
+describe('Messaging', () => {
+    beforeEach(() => {
+        // # Create new user that will eventually be inactive
+        cy.apiLogin('sysadmin');
+        cy.apiCreateNewUser().then((user) => {
+            cy.wrap(user).as('userInactive');
+        });
+    });
+
+    it('M23349 - Deactivated users are not shown in Direct Messages modal', () => {
+        // # Deactivate "user-2"
```

Comment:
```suggestion
        // # Deactivate inactive user
```
Applicable to all that menioned "user-2"